src/machine/machinectl.c

```diff
@@ -1119,20 +1119,38 @@ static int copy_files(int argc, char *argv[], void *userdata) {
                 host_path = abs_host_path;
         }
 
-        r = bus_message_new_method_call(
-                        bus,
-                        &m,
-                        bus_machine_mgr,
-                        copy_from ? "CopyFromMachine" : "CopyToMachine");
-        if (r < 0)
-                return bus_log_create_error(r);
+        if (arg_force) {
```

Comment:
We can just always use the "WithFlags" methods in machinectl. The command line tools don't need to support older versions of the daemons. If --force is not specified, you can just pas "0" as the flags argument.

Reply:
Oh, actually, that's not really true, we *do* prefer to use the old methods, because people use -M or -H to access remote systems or container environments.

Hence, it's *good* to use the old calls when we can.

Reply:
So to be explicit, is it preferred that I use the "CopyFromMachine" method when no flags are necessary, and use "...WithFlags" only when there are flags?